APTITUDE/APTITUDETEST.Data/Data/QuestionRepository.cs

```diff
@@ -231,10 +231,28 @@ public async Task<JsonResult> Create(QuestionVM questionVM)
 
             try
             {
-                if (questionVM.Id != 0 || !ValidateQuestion(questionVM) || !ValidateOptionText(questionVM) || !ValidateOptionImages(questionVM))
+                List<int> duplicateOptionIds = new List<int>();
+                List<QuestionOptions> duplicateOptions = new List<QuestionOptions>();
+                if (questionVM.Id != 0 || !ValidateQuestion(questionVM) || !ValidateOptionText(questionVM) || (questionVM.DuplicateFromQuestionId == 0 && !ValidateOptionImages(questionVM)))
                 {
                     return new JsonResult(new ApiResponse<string>() { Message = ResponseMessages.BadRequest, Result = false, StatusCode = ResponseStatusCode.BadRequest });
                 }
+                if (questionVM.DuplicateFromQuestionId != 0)
+                {
+                    Question duplicateQuestion = _context.Questions.Where(x => x.Id == questionVM.DuplicateFromQuestionId).FirstOrDefault();
+                    if (duplicateQuestion.OptionType != (int)Enums.OptionType.ImageType && questionVM.OptionType == (int)Enums.OptionType.ImageType && !ValidateOptionImages(questionVM))
```

Comment:
check duplicateQuestion is null

Reply:
Code added for checking null, will commit in next pr as this is merged